src/executables/master.c

```diff
@@ -444,26 +444,9 @@ receive_server_info (CSS_CONN_ENTRY * conn, unsigned short rid, std::string & db
   int exit_code = css_receive_data (conn, rid, &buffer, &buffer_length, -1);
   if (exit_code == NO_ERRORS)
     {
-      const char first_char = buffer[0];
-      if (first_char == '#')
-	{
-	  // HA mode on; only transaction servers connect.
-	  type = SERVER_TYPE_TRANSACTION;
-	  // Include '#' in the dbname; legacy requirement
-	  dbname = std::string (buffer, buffer_length);
-	}
-      else if (first_char == '$' || first_char == '%')
-	{
-	  // not really a server, it is copylogdb or applylogdb
-	  type = SERVER_TYPE_UNKNOWN;
```

Comment:
hmm.. I think you should keep it if you want to keep copylogdb and applyogdb code. Or you should delete others like in `css_process_all_list_info`. What do you think?